packages/components/Badge/src/Badge.types.ts

```diff
@@ -70,6 +77,12 @@ export interface BadgeProps extends BadgeCoreProps, BadgeConfigurableProps {
    * @defaultvalue filled
    */
   appearance?: BadgeAppearance;
+
+  /**
+   * Applies shadow according to the Fluent elevation system.
+   * @default false
+   */
+  shadow?: boolean;
```

Comment:
If we keep this prop could consider renaming to shouldShowShadow or something similar

Reply:
I removed it